proposals/templates/proposals/proposal_form.html

```diff
@@ -101,11 +101,72 @@ <h2>{% trans "Algemene informatie over de aanvraag" %}</h2>
                 {% endblocktrans %}
                 <br/>
                 <br/>
-            {% endif %}
+		{% endif %}
             <form action="" method="post" enctype="multipart/form-data">{% csrf_token %}
                 <table>{{ form.as_table }}</table>
                 {% include "base/form_buttons.html" %}
             </form>
         </div>
     </div>
+    <!-- Code for intended start date warning -->
+    <style>
+     .red {
+	 color: red;
+     }
+     #date_start_warning {
+	 margin-top: 0.5rem;
+	 margin-bottom: 0.5rem;
+	 line-height: 1.5;
+     }
+    </style>
+    <script>

```

Comment:
Indentation in this script block could use some work. Everything seems to be on the same indentation level

Reply:
Also, you're somewhat inconsistat with your semicolors... I don't mind it that much, but consistancy is nice

Reply:
I added some more semicolons for good measure